setup.py

```diff
@@ -14,13 +14,15 @@
 
 from setuptools import setup, find_packages
 
-__version__ = None
-_minimum_jaxlib_version = '0.1.62'
+# The following should be updated with each new jaxlib release.
```

Comment:
Can you please add a comment to `jax/lib/version.py` explaining that this must be done?

Or alternatively could we somehow avoid the redundancy?

Reply:
I did add this as a step in our release playbook.

I'm not sure it's possible to remove any redundancy, because the current jaxlib version is not available anywhere else in the files associated with a release (the `jaxlib` directory is not included in the `jax` tarball), and the CUDA versions used to build the released wheels are a property of the repo at build time, not the repo at HEAD.

Reply:
A comment in the file is certainly possible. I'm more likely to see it if it's right next to the version number.

However, note that `jax/lib/` (N.B. not `jaxlib`) probably *is* in our `jax` tarball. It gets copied into `jaxlib` by the build script, but it's probably present in both.

Reply:
I misremembered slightly. The `version.py` file is in `jaxlib/`. But it doesn't have to be that way. We could easily include it in `jax` as well, you'd just have to modify the wheel build script to reflect the new location.

Reply:
Added a comment to `jaxlib/version.py`. I think that's our best fix for now.